src/index.html

```diff
@@ -11,7 +11,173 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset class="fieldset-block">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input type="text"
+                   name="lastname"
+                   autocomplete="off"
+                   class="input-margin"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input type="text"
+                   name="firstname"
+                   autocomplete="off"
+                   class="input-margin"
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="firstname"
            autocomplete="off"
            class="input-margin"
          >
```

![image](https://user-images.githubusercontent.com/80700247/233829292-0ca9e06b-af0f-4020-9123-05dc3ccf8201.png)